src/Amadeus/Client/Struct/Fare/MasterPricer/FeeId.php

```diff
@@ -33,4 +33,20 @@ class FeeId
     public $feeType;
```

Comment:
`$feeType` and `$feeIdNumber` need phpdocs.

Also here I'd like to have the same list of constants of possible feeTypes.